lib/article_json/elements/paragraph.rb

```diff
@@ -32,6 +32,16 @@ def blank?
         end
       end
 
+      # Return the sum of all characters within the content's text elements
+      # @return [Integer]
+      def length
+        return 0 if empty?
+        @content.reduce(0) do |sum, element|
+          sum + (element.respond_to?(:length) ? element.length : 0)
```

Comment:
You could also write this as `sum + (element&.length || 0)`

Reply:
Sadly this won't work because the safe navigation operator `&.` works different than `ActiveSupport`'s `#try`: it only checks if the expression before evaluates to `nil`, if not it executes the provided method.

Example:
```ruby
require 'active_support'

class MyClass
  def my_method
    :hello_world
  end
end

my_instance = MyClass.new

my_instance.try(:my_method)
# => :hello_world

my_instance&.my_method
# => :hello_world

my_instance.try(:unknown_method)
# => nil

my_instance&.unknown_method
# => NoMethodError: undefined method `unknown_method' for  #<MyClass:0x007fd182884108>


```

Reply:
True, I obviously didn't think that one through :D